Sky Atlas/Sky Atlas.html

```diff
@@ -16384,63 +16376,243 @@ <h1>Sections & Primary Docs</h1>
         </tr>
         <tr>
           <td>
-            <dfn>A.4.4
-              - A1 - Activation And Sealing Mechanisms - SKY Activation</dfn>
+            <dfn>A.4.4 - A1 - SKY Staking Mechanism - Sky Staking</dfn>
           </td>
-          <td>Activation And Sealing Mechanisms - SKY Activation</td>
+          <td>SKY Staking Mechanism</td>
           <td>Section</td>
-          <td>SKY Activation can be done by any SKY holder through the SKY Activation Mechanism available on Ethereum Mainnet and SkyLink Deployments. SKY Activation provides USDS Rewards or Agent Token Rewards.</td>
+          <td>SKY holders can stake their tokens via the SKY Staking Mechanism available on Ethereum Mainnet and SkyLink Deployments. SKY stakers earn voting rewards sourced from the Sky Treasury Management Function. In lieu of USDS rewards, SKY stakers can choose to earn Agent Token Rewards, including SPK and the tokens of other Star Agents incubated by the Launch Project. SKY stakers can also borrow USDS against their staked collateral using the SKY-backed borrowing mechanism defined herein.</td>
         </tr>
         <tr>
           <td>
-            <dfn>A.4.4 - Activation And Sealing Mechanisms - SKY Activation - Availability</dfn>
+            <dfn>A.4.4 - A1 - SKY Staking Mechanism - SKY Staking - SKY Staking Voting Rewards</dfn>
```

Comment:
You only need "A1", "A2" etc for the Sections
```suggestion
            <dfn>A.4.4 - SKY Staking Mechanism - SKY Staking - SKY Staking Voting Rewards</dfn>
```